packages/packages.csv

```diff
@@ -156,7 +156,7 @@ Microsoft.Extensions.Configuration.Abstractions,ship,include
 Microsoft.Extensions.Configuration.AzureKeyVault,ship,include
 Microsoft.Extensions.Configuration.Binder,ship,include
 Microsoft.Extensions.Configuration.CommandLine,ship,include
-Microsoft.Extensions.Configuration.DockerSecrets,ship,include
+Microsoft.Extensions.Configuration.DockerSecrets,noship,include
```

Comment:
Functionally, we aren't even really making use of the include/exclude flag right now since it's all controlled by what's going into the MetaPackage reference. There's is a work item https://github.com/aspnet/MetaPackages/issues/19 to eventually generate what we have in the MetaPackage based on this file so it's a good idea to keep it up to date as if it were used to generate the runtime store.